src/plugins/intel_cpu/tests/functional/utils/fusing_test_utils.hpp

```diff
@@ -93,6 +94,7 @@ class CpuTestWithFusing : public CPUTestsBase {
     std::shared_ptr<postOpMgr> postOpMgrPtr;
     std::vector<std::string> fusedOps;
     bool checkFusingPosition = true;
+    bool isFused = true;
```

Comment:
Maybe rename to "expectPostOpsToBeFused" for better understanding?

Reply:
fixed